css/localgov-alert-banner.css

```diff
@@ -9,14 +9,30 @@
 .localgov_alert_banner--inner {
   display: flex;
   justify-content: space-between;
-  margin: 0rem 0.9375rem;
-  padding: 0.625rem 0rem;
+  margin: 0 0.9375rem;
+  padding: 0.625rem 0;
 }
 .localgov_alert_banner--actions {
   display: flex;
   justify-content: space-between;
   align-items: flex-end;
 }
+.localgov_alert_banner--actions button {
+  background: #77bb55;
+  border: none;
+  color: #000;
+  font-weight: bold;
+  padding: 0.4rem 0.8rem;
```

Comment:
Is adding the button colours intentional here? That would be better placed for a theme to handle. If our aim is to have the button not look like a system button then it might be a good idea to add the following here and do a basic style like a link. If not best to leave as a button and in the theme apply the right styling. 
````css
-webkit-appearance: none;
-moz-appearance: none;
appearance: none;
````
I

Reply:
Thanks @andybroomfield that makes sense. Have removed the button styling and left with system styling, looking at the localgov_theme I can now see that's where this should be handled.